### PR TITLE
Multi-Line Arguments Fix

### PR DESCRIPTION
**Pull request by Harsh188**
No related PR.

### Changes made to:
- [text_classification.ipynb](https://www.tensorflow.org/tutorials/keras/text_classification)

### Description:
The [loss function and optimizer](https://www.tensorflow.org/tutorials/keras/text_classification#loss_function_and_optimizer) `model.compile` function is hard to read since the arguments are not split into multiple lines.

### The following changes were made:
```python
model.compile(loss=losses.BinaryCrossentropy(from_logits=True), optimizer='adam', metrics=tf.metrics.BinaryAccuracy(threshold=0.0))
```

to

```python
model.compile(loss=losses.BinaryCrossentropy(from_logits=True),
                          optimizer='adam',
                          metrics=tf.metrics.BinaryAccuracy(threshold=0.0))
```

### CODE REVIEW

**Comment by lamberta** [comment]
Please remove the notebook outputs, thanks
`python3 -m tensorflow_docs.tools.nbfmt --remove_outputs notebook.ipynb`

**Review by MarkDaoust — APPROVED** [review]
